src/aap_eda/utils/__init__.py

```diff
@@ -26,5 +26,5 @@ def get_package_version(package_name: str) -> str:
     try:
         return importlib.metadata.version(package_name)
     except importlib.metadata.PackageNotFoundError:
-        logger.error("Cannot read version from %s package", package_name)
+        logger.warning("Cannot read version from %s package", package_name)
```

Comment:
IMO error is correct here.

Reply:
We expect a version and it is not there. There is a real loss because we don't report the version. To me is an error. But I'm not going to waste my bullets on this either. Up to you.

Reply:
> We expect a version and it is not there.

Or even worse: it tries to query for a package that is not installed at all.

Reply:
rolling back to error